### ISSUE TITLE
How did you differentiate between the test set and the training set?

### ISSUE THREAD

**Issue by JUNEeer** [oppo-us-research/SpacetimeGaussians#93]

**Comment by lizhan17** [comment]
We differ training and testing cameras following previous methods, picking one camera from whole camera sets.  Usually it is a center camera in a camera array.

**Comment by JUNEeer** [comment]
> We differ training and testing cameras following previous methods, picking one camera from whole camera sets. Usually it is a center camera in a camera array.

Do you mean that each camxx data set is a separate test set or training set?

**Comment by lizhan17** [comment]
no, just one camera is picked. all the videos of that camera are test set.

**Comment by JUNEeer** [comment]
> no, just one camera is picked. all the videos of that camera are test set.

I don’t quite understand it. Can you explain it in detail with an example?

**Comment by lizhan17** [comment]
if the cam frame dict is like 

cam1: frame0,1,2,3...
cam2: frame0,1,2,3...
cam3: frame0,1,2,3...
cam4: frame0,1,2,3...

cam1: frame0,1,2,3... are tested

the rest are training sets.
